api/src/main/java/module-info.java

```diff
@@ -93,36 +95,39 @@
- * specification to define entities for relational databases.
- * You may use {@code jakarta.nosql.mapping.Entity} and the corresponding
- * entity-related annotations of the Jakarta NoSQL specification to define
- * entities for NoSQL databases. For other types of data stores, you may use
- * other entity models that are determined by the Jakarta Data provider
- * for the respective data store type.</p>
+ * <p>Jakarta Persistence and Jakarta NoSQL define programming models for entity
+ * classes that may be used with Jakarta Data:</p>
+ * <ul>
+ * <li>{@code jakarta.persistence.Entity} and the corresponding entity-related
+ *    annotations of the Jakarta Persistence specification may be used to
+ *    define entities stored in a relational database, or</li>
+ * <li>{@code jakarta.nosql.mapping.Entity} and the corresponding entity-related
+ *     annotations of the Jakarta NoSQL specification may be used to define
+ *     entities stored in a NoSQL database.
+ * </ul>
+ * <p>A Jakarta Data provider may define its own programming model for entity
+ *    classes representing some other arbitrary kind of data.</p>
  *
  * <p>Methods of repository interfaces must be styled according to a
- * defined set of conventions, which instruct the container/runtime
+ * well-defined set of conventions, which instruct the container/runtime
  * about the desired data access operation to perform. These conventions
- * consist of patterns of reserved keywords within the method name,
- * method parameters with special meaning, method return types,
- * and annotations that are placed upon the method and its parameters.</p>
- *
- * <p>Built-in repository super interfaces, such as {@link DataRepository},
- * are provided as a convenient way to inherit commonly used methods and are
- * parameterized with the entity type and id type. Other built-in repository
- * interfaces, such as {@link BasicRepository}, can be used in place of
- * {@link DataRepository}
- * and provide a base set of predefined repository methods
- * which serve as an optional starting point.
- * You can extend these built-in interfaces to add your own custom methods.
- * You can also define your own repository interface without inheriting from the
- * built-in super interfaces. You can copy individual method signatures from the
- * built-in repository methods onto your own, which is possible
- * because the built-in repository methods are consistent with the
- * same set of conventions that you use to write custom repository methods.</p>
+ * consist of patterns of reserved keywords within the method name, method
+ * parameters with special meaning, method return types, and annotations
+ * placed upon the method and its parameters.</p>
+ *
+ * <p>Built-in repository superinterfaces, such as {@link DataRepository},
+ * are provided as a convenient way to inherit commonly used methods, and
+ * are parameterized with the entity type and by its id type. Other built-in
```

Comment:
minor grammar - I think the comma might not be correct here
```suggestion
 * are provided as a convenient way to inherit commonly used methods and
 * are parameterized with the entity type and by its id type. Other built-in
```